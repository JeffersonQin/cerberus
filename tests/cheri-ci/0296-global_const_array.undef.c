int xs[2];
const int ys[2];

int main(void)
{
  xs[0] = 1; // this is ok
  ys[0] = 1; // but this is UB
}
