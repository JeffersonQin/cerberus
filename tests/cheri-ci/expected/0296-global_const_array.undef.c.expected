Undefined {ub: "UB064_modifying_const", stderr: "", loc: "<7:3--7:12>"}
